meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
I very much like this idea, I did not know how to do something like this via local.conf. As I feel there is no urgency to merge this one, I'd propose to keep this open and I'll figure out how to make this simpler for customers. Would it be OK to do it as follows?
- Add fragment to meta-ostro-bsp/recipes-kernel/linux-yocto/linux-yocto
- Add #SRC_URI_append = " file://bootchart.cfg" to local.conf.sample